src/pages/Swap/LimitOrderBox/utils/index.ts

```diff
@@ -0,0 +1,61 @@
+import { AddressZero } from '@ethersproject/constants'
+import { formatUnits } from '@ethersproject/units'
+import { ChainId, Price, Token, TokenAmount, USDC } from '@swapr/sdk'
+
+import createDebugger from 'debug'
+
+import { isAddress } from '../../../../utils'
+import { LimitOrderKind, SerializableLimitOrder } from '../interfaces'
+import { TokenAmount as ITokenAmount } from '../interfaces/token.interface'
+
+/**
+ *
+ * @param amount The amount of tokens to be bought or sold
+ * @returns
+ */
+export const formatTokenAmount = (amount: ITokenAmount) => formatUnits(amount.amount, amount.token.decimals)
+
+export const debug = createDebugger('limit-orders')
+
+interface InitialState {
+  sellTokenAmount: TokenAmount
+  buyTokenAmount: TokenAmount
+  price: Price
+  limitOrder: SerializableLimitOrder
+}
+
+/**
+ *
+ * @param chainId The chain id of the network
+ */
+export function getInitialState(chainId: ChainId, account: string): InitialState {
+  const sellTokenAmount = new TokenAmount(Token.getNativeWrapper(chainId), '1000000000000000000')
+  const buyTokenAmount = new TokenAmount(USDC[chainId], '0')
```

Comment:
Any specific reason for USDC and not DAI? Just feels weird to swap tokens between Swap and Limit UI

Reply:
DAI, ETH and all default tokens are not available in limit orders.